lib/scss/component/views/stanford_publication.scss

```diff
@@ -1,8 +1,111 @@
 @charset "UTF-8";
 
+@import '../config';
+
+.stanford-publications-related {
+  @include responsive-spacing('margin-bottom', 9);
+
+  .block__title {
+    @include type-b;
+
+    text-align: center;
+  }
+
+  .flex-3-of-12 {
+    @include grid-media-min('lg'){
+      flex: calc(33% - 36px);
+      max-width: calc(33% - 36px);
+    }
+
+    @include grid-media-max('md'){
+      flex: calc(100% - 36px);
+      max-width: calc(100% - 36px);
+    }
+  }
+
+
+  .su-card__contents {
+
+    span {
+      @include small-paragraph;
+
+      text-transform: uppercase;
+    }
+
+    a {
+      @include small-paragraph;
+
+      font-weight: $su-font-regular;
+      color: $su-color-bright-red;
+      text-decoration: none;
+
+      &:hover,
+      &:focus {
+        color: $su-color-black;
+        text-decoration: underline;
+      }
+    }
+
+
+    h2 {
+      > a {
+        @include type-e;
+        @include link-icon(caret-right, 20px, right);
```

Comment:
Did you get this working? what was the solution?

Reply:
Instead of adding the styles to the view, it wasn't working the way I wanted to.  I added the icon in the css.  The problem with the webpack was there was a difference in the location of the actual file.